src/App.tsx

```diff
@@ -1,18 +1,32 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
-  .map(n => `Item ${n}`);
+const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [perPage, setPerPage] = useState(5);
+  const [page, setPage] = useState(1);
+
+  const firstItemIndex = (page - 1) * perPage;
+  const lastItemIndex = firstItemIndex + perPage;
+  const currentItems = items.slice(firstItemIndex, lastItemIndex);
+
+  const totalItems = items.length;
+  const lastItem = currentItems[currentItems.length - 1].slice(-2).trim();
```

Comment:
Using the last item's text to get the last item index may cause issues if the item text format changes. It's better to calculate it using the page and perPage variables.